test/sequential/test-inspector-port-zero.js

```diff
@@ -34,7 +34,10 @@ function test(arg, port = '') {
     };
     proc.stdout.on('close', mustCall(() => onclose()));
     proc.stderr.on('close', mustCall(() => onclose()));
-    proc.on('exit', mustCall((exitCode) => assert.strictEqual(exitCode, 0)));
+    proc.on('exit', mustCall((exitCode, signal) => assert.strictEqual(
+      0,
+      exitCode,
+      `exitCode was ${exitCode} not 0, signal was ${signal}`)));
```

Comment:
"not 0" is tautological here.

As well, the order of arguments is `actual, expected[, message]`.  You flipped them around here.

Reply:
>"not 0" is tautological here.

True, just trying to save someone the trouble of having to look at the source. But thinking again, I think shorter is better.

>As well, the order of arguments is actual, expected[, message]. You flipped them around here.

Huh, I always thought it was `expected, actual`, guess I have to relearn that.

Both fixed.